packages/material-tailwind-react/src/components/Alert/index.tsx

```diff
@@ -55,8 +55,7 @@ export interface AlertProps extends Omit<MotionProps, "animate"> {
 export const Alert = React.forwardRef<HTMLDivElement, AlertProps>(
   ({ variant, color, icon, open, action, onClose, animate, className, children, ...rest }, ref) => {
     // 1. init
-    const { alert } = useTheme();
-    const { defaultProps, valid, styles } = alert;
+    const { defaultProps, valid, styles } = useTheme("alert");
```

Comment:
why this change across for usage of useTheme()?

Reply:
By default library was using all components' theme constant resulting in 20Kb of constant just loaded where we are using less than 1kB, so migrated the logic to inject the theme via theme-provider on-demand.
For new components, If the theme is not injected, we will throw an error so the developer can add the theme & start using it, which would ease the load on the root level.